Documentation/openssl.md

```diff
@@ -29,7 +29,7 @@ $ openssl genrsa -out ca-key.pem 2048
 $ openssl req -x509 -new -nodes -key ca-key.pem -days 10000 -out ca.pem -subj "/CN=kube-ca"
 ```
 
-You need to store the CA keypair in a secure location for future use.
+**It is highly recommended to generate and store keypairs in a secure location for future use, i.e. local machine.**
```

Comment:
I would probably argue the opposite, that the deployer's local machine is not the best place to store this data. Ultimately this is provided as a proof-of-concept, but I don't think we should be calling out the deployer's local machine as the "highly recommended secure location" for key material.